components-api/src/main/scala/pl/touk/nussknacker/engine/api/generics/ExpressionParseError.scala

```diff
@@ -16,23 +16,18 @@ class GenericFunctionError(messageInner: String) extends ExpressionParseError {
 }
 
 
-sealed abstract class Signature {
-  def display: String
-
-  protected def typesToString(types: List[TypingResult]): String =
+final class Signature(val name: String, val noVarArgs: List[TypingResult], val varArg: Option[TypingResult]) {
```

Comment:
Can't we do the same with `VarArgsMethodInfo` and `NoVarArgsMethodInfo`?

Reply:
:+1:

Reply:
I have already done it, but its with adding support for varArgs in generic functions